jwst/regtest/test_miri_lrs_dedicated_mbkg.py

```diff
@@ -0,0 +1,40 @@
+import os
+import pytest
+from astropy.io.fits.diff import FITSDiff
+
+from jwst.pipeline.collect_pipeline_cfgs import collect_pipeline_cfgs
+from jwst.stpipe import Step
+
+
+@pytest.fixture(scope="module")
+def run_pipeline(jail, rtdata_module):
+
+    rtdata = rtdata_module
+
+    collect_pipeline_cfgs("config")
+
+    rtdata.get_asn("miri/lrs/miri_lrs_mbkg_dedicated_spec3_asn.json")
+
+    args = ["config/master_background.cfg", rtdata.input,
+            "--save_results=True"]
+    Step.from_cmdline(args)
+
+    return rtdata
+
+
+@pytest.mark.bigdata
+def test_miri_lrs_dedicated_mbkg(run_pipeline, fitsdiff_default_kwargs):
+    """Run a test for MIRI LRS data with dedicated background exposures."""
+
+    # Run the step and retrieve the output
```

Comment:
This comment is misleading.  The running of the pipeline actually happens on test setup and the it returns a `RegtestData` object to this variable.  Here it is just assigning it to a different name.